include/Evaluation_und_Validation.tex

```diff
@@ -205,40 +228,79 @@ \subsection{Evaluation}
+		\label{tbl:distanceCNN}
+	\end{table}
+	\begin{table}[H]
+		\centering
+		\scriptsize
+		\begin{tabularx}{.9\textwidth}{Xrrrrrr}
+			\hline
+			\multicolumn{7}{c}{\textbf{Threshold-Methode Ergebnisse Distanz Experiment}}\\
+			\hline
+			\textbf{Metriken} & \textbf{10cm} & \textbf{20cm} & \textbf{30cm} & \textbf{40cm} & \textbf{50cm} & \textbf{60cm}\\
+			\hline 
+			\textbf{Gesamtanzahl Personen in Bilder} & 29 & 18 & 54 & 18 & 51 & 28\\
+			\hline
+			\textbf{Korrekt identifizierter Personen} & 22 & 15 & 54 & 18 & 51 & 28\\
+			\hline
+			\textbf{Falsche Treffer} & 4 & 1 & 3 & 1 & 3 & 1\\
+			\hline
+			\textbf{Nicht erkannte Personen} & 7 & 3 & 0 & 0 & 0 & 0\\
+			\hline
+			\textbf{Recall} & 75.9\% & 83.3\% & 100\% & 100\% & 100\% & 100\%\\
+			\hline  
+			\textbf{Precision} & 84.6\% & 93.8\% & 94.7\% & 94.7\% & 94.4\% & 100\%\\
+			\hline
+			\textbf{F1-Score} & 80.0\% & 88.2\% & 97.2\% & 97.2\% & 97.1\% & 100\%\\
+			\hline
+		\end{tabularx}
+		\caption{Ergebnisse, der Threshold-Methode, der Distanz Experimente}
+		\label{tbl:distanceThresh}
+	\end{table}
+}
 
 \subsection{Evaluation}
 
-Die Algorithmen können Personen bereits ab einem Abstand von 10cm auseinanderhalten, sofern die Personen so ausgerichtet sind, dass die optische Verzerrung keinen Einfluss auf den Zwischenraum hat. Wie in den Abbildungen in der linken Bildhälfte zu sehen ist, können die Personen, die auf die Kamera ausgerichtet sind, unterschieden werden. Sind die Person jedoch so platziert, wie auf der rechten Bildhälfte zu sehen ist, sind diese von der Kamera aus gesehen, hintereinander. Bei so kleinen Abstand kann der Algorithmus deshalb nicht mehr herausfiltern, ob es sich um eine oder mehrere Personen handelt.\\
-\\
-Befinden sich die Personen in der schlechtest möglichen Position, benötigen die Algorithmen einen Mindestabstand von 30cm, um die Personen fehlerfrei unterscheiden zu können. Siehe Abbildungen \ref{fig:cnnDistance30} und \ref{fig:thresholdDistance30}
+
+Das \gls{CNN} Kann bereits ab einem Abstand von 10cm zuverlässig Personen auseinanderhalten.\\
+Die Threshold-Methode kann Personen bereits ab einem Abstand von 10cm auseinanderhalten, sofern die Personen so ausgerichtet sind, dass die optische Verzerrung keinen Einfluss auf den Zwischenraum hat. Wie in der Abbildung \ref{fig:thresholdDistance10} in der linken Bildhälfte zu sehen ist, können die Personen, die auf die Kamera ausgerichtet sind, unterschieden werden. Sind die Person jedoch so platziert, wie auf der rechten Bildhälfte zu sehen ist, sind diese von der Kamera aus gesehen, hintereinander. Bei so kleinen Abstand kann der Algorithmus deshalb nicht mehr herausfiltern, ob es sich um eine oder mehrere Personen handelt.\\
+Befinden sich die Personen in der schlechtest möglichen Position, benötigt die Threshold-Methode einen Mindestabstand von 30cm, um die Personen fehlerfrei unterscheiden zu können (siehe Abbildung \ref{fig:thresholdDistance30}).
```

Comment:
Befinden sich die Personen in der **schlechtmöglichsten** Position, [...]

Reply:
*schlechtestmöglichen* gemäss LT